mudlet-mapper.xml

```diff
@@ -6739,6 +6739,15 @@ end
   return c
 end
 
+function mmp.changeEchoColour()
+  if color_table[mmp.settings.echocolour] then
+    mmp.echo(&quot;Echo colour is now &quot;..mmp.settings.echocolour)
+  else
+    mmp.echo(&quot;Not a valid cecho color - Setting to default&quot;)
+    mmp.settings:setOption(&quot;echocolour&quot;, &quot;MediumBlue&quot;)
```

Comment:
Defaulting to a lighter blue seems reasonable, given how many people have issues with the current blue on certain monitors with the gamma too dark.